packages/lesswrong/components/posts/PostsPage/PostsCoauthor.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { useCurrentUser } from '../../common/withUser';
+import { registerComponent, Components } from '../../../lib/vulcan-lib';
+import { postCoauthorIsPending } from '../../../lib/collections/posts/helpers';
+
+const PostsCoauthor = ({ post, coauthor }: {
+  post: PostsDetails,
+  coauthor: UsersMinimumInfo,
+}) => {
+  const isPending = postCoauthorIsPending(post, coauthor._id);
+
+  const currentUser = useCurrentUser();
+  if (isPending && currentUser?._id !== post.userId) {
+    return null;
+  }
```

Comment:
What if I'm a coauthor?

Reply:
If you're a coauthor then you'll see the box that says "This user has requested you to co-author this post" with accept and decline buttons. I felt like that was probably enough without showing it here as well, but maybe not?

Reply:
Yep. But, like, not that important.

Reply:
Sounds good! I've just pushed a commit which changes this and also fixes all the other comments you made on the PR.